roles/docker/tasks/cleanup-rpm.yml

```diff
@@ -0,0 +1,18 @@
+- name: install docker-cleanup package
+  sudo: yes
+  yum:
+    name: "https://dl.bintray.com/asteris/mantl-rpm/docker-cleanup-0.0.1-1.noarch.rpm"
+    state: present
+  tags:
+    - docker
+
+- name: add default container exclusions to docker cleanup
+  sudo: yes
+  lineinfile:
+    dest: /etc/docker-cleanup/docker-gc-exclude-containers
+    line: "{{ item }}"
+  with_items:
+    - "# this is in an EXIT status most of the time, but is needed intact"
+    - "{{ zookeeper_data_volume }}"
```

Comment:
Let's add an empty list of excluded containers to the defaults.yml file, this would almost definitely be useful for people using mantl.